sebtaskAPI/sebtaskAPI/Services/EcbService.cs

```diff
@@ -0,0 +1,132 @@
+using sebtaskAPI.Data;
+using sebtaskAPI.Models;
+using System.Data.Entity;
+using System.Globalization;
+using System.Net.Http;
+using System.Xml;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace sebtaskAPI.Services
+{
+  public class EcbService
+  {
+    private readonly DataContext _dataContext;
+
+    public EcbService(DataContext dataContext)
+    {
+      _dataContext = dataContext;
+    }
+
+    public ExchangeRate? GetRateByCurrencyCode(string code) {
+
+      return _dataContext.ExchangeRates.Where(c=> c.Currency == code).FirstOrDefault();
+    }
```

Comment:
This method could be `async` and we could use `FirstOrDefaultAsync` instead of `Where` method.